app/_src/guides/progressively-rolling-in-strict-mtls.md

```diff
@@ -175,13 +176,15 @@ flowchart LR
 
 Finally, to set strict mode you can either edit the policy or remove it (the default is taken from `Mesh` object which is `STRICT`).
 
-```bash
-kubectl delete meshtlses.kuma.io -n kuma-system redis
-```
+{% tip %}
+**Things to remember when migrating to strict TLS**
 
-## Things to remember when migrating to strict TLS
+Before changing a workload to `Strict` mode check that `tls_inspector.tls_not_found` stat is no longer incrementing.
```

Comment:
we no longer use this stat to indicate this, we're using https://github.com/kumahq/kuma-website/pull/1967/files#diff-2f2247fada7fc4374636439b1747188dfc75fb57ae927de5bd76780696012f4dR101 so it should be something like the difference between `inbound_POD_IP_6379.rbac.allowed: 809` and `cluster.localhost_6379.upstream_cx_total` from up the point of the migration should  be 0. Not sure how to clearly indicate that.

Reply:
We also sometimes say metric and sometimes stat. Which I think is probably pretty confusing, it should be always pointed out how to get this value. Since stats are not used in almost all mesh operations